regression/test.pl

```diff
@@ -33,7 +33,12 @@ sub with_color {
 
 sub run($$$$$) {
   my ($name, $input, $cmd, $options, $output) = @_;
-  my $cmdline = "$cmd $options '$input' >'$output' 2>&1";
+  my $cmdline;
```

Comment:
:pick: Typo in commit message for commit `Do not pass empty commandline argument in test.pl `: 
esample -> example